src/components/input/input.stories.mdx

```diff
@@ -33,3 +41,64 @@ You can also add `c-input` to `<select>` elements, in which case you’ll get a
     })}
   </Story>
 </Preview>
+
+## Elastic textarea
+
+Add `js-elastic-textarea` to a `<textarea>` element\* to have it expand as the user types more than will fit.
```

Comment:
Is this accurate? It seems like the class name you added would be reliant on the JavaScript you write? e.g. you might want `.js-comment-field` to be the class?

Reply:
Maybe "Call `createElasticTextArea(yourTextareaSelector)` to have your textarea expand as the user types more than will fit." or something like that?

Reply:
Oh, good points, @Paul-Hebert!

Reply:
Just want to make a note, the createElasticTextArea function accepts an element, not a selector. But I agree with this suggestion otherwise

Reply:
Oh, right. Good call!

Reply:
You're correct with regards to the JS, but I'm actually about to add some elastic textarea-specific CSS, so they'll need this specific class. I'll add a note to that regard.

Reply:
I reconsidered, and updated the documentation to differentiate between the classname needed for styling and the one for JS selection, and indicated that you could use any classname for selection.

Reply:
> Just want to make a note, the createElasticTextArea function accepts an element, not a selector. 

How did the previous JS Class version work?

I would expect to pass a CSS selector to the `createElasticTextArea()` function. But I also don't recall how the previous version was working. 🤷‍♀️

Reply:
Or, wait...I'm getting myself all confused. Ignore my comment for now. I think you'll have it handled. LOL